bootstrap/generate_bootstrap_plans

```diff
@@ -0,0 +1,23 @@
+nix build nixpkgs#jq.bin -o jq
```

Comment:
The bootstrap directory has a README. Would it be possible to document this script and its operation there?

Also, historically there was a lot of pushback against adding anything nix-related to this repository. I personally love nix and happy to have it. Just wanted to make a note of the precedents.

Reply:
Without this script there is no way to update the bootstrap plans. You don't have to use it.. I copied the script from the one we have used in GHC to update our bootstrap plans.

Reply:
This is a great addition to cabal's infrastructure. But as usual, without docs, it has very high chance to be overlooked / bit-rot in the future. I think it'd be very helpful to add a line about it in the README.

Reply:
And what does https://github.com/haskell/cabal/blob/master/Makefile#L210 do? It was a while since I used it...

Reply:
@ulysses4ever I had added something to the README like you suggested, did you have something else in mind?